src/compas_fab/robots/wrench.py

```diff
@@ -373,7 +373,7 @@ def gravity_compensated(self, ft_sensor_frame, mass, center_of_mass):
         ----------
         .. [1] Vougioukas S., *Bias Estimation and Gravity Compensation For
             Force-Torque Sensors*,
-            Available at: https://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.552.109
+            Available at: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.552.109
```

Comment:
how lame can they be to not have ssl in 2022? :P

Reply:
Lol they have but their certificate expired about a week ago..